### PR TITLE
improve theme

### PR DESCRIPTION
**Pull request by aaranadev**

### CODE REVIEW

File: template-skeleton/template/package.dev.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@turf/boolean-intersects": "^6.3.0",
     "cypress": "^6.3.0",
     "deck.gl": "8.5.0-alpha.9",
+    "deepmerge": "^4.2.2",
```
Comment (alasarr) [inline template-skeleton/template/package.dev.json:31]: Why this?

File: template-skeleton/template/package.dev.json [hunk 2]
```diff
@@ -43,13 +44,13 @@
...
     "firebase-tools": "^9.2.2",
-    "start-server-and-test": "^1.12.0",
     "husky": "^4.3.8",
     "hygen": "^6.1.0",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
     "react-app-rewired": "^2.1.8",
-    "react-is": "^17.0.1"
+    "react-is": "^17.0.1",
+    "start-server-and-test": "^1.12.0"
```
Comment (alasarr) [inline template-skeleton/template/package.dev.json:53]: Why this?

**Review by alasarr — CHANGES_REQUESTED** [review]

File: template-skeleton/template/package.dev.json (hunk 2 above)
Comment (aaranadev) [reply template-skeleton/template/package.dev.json:53]: `start-server-and-test` is already installed from before

File: template-skeleton/template/package.dev.json (hunk 1 above)
Comment (aaranadev) [reply template-skeleton/template/package.dev.json:31]: If a user (PS) wants to edit the theme, it is possible to do so without breaking it.
```js
cartoThemeOptions = {
  pallete: {
    main: '#fff
}

```

File: template-skeleton/template/package.dev.json (hunk 2 above)
Comment (alasarr) [reply template-skeleton/template/package.dev.json:53]: What do you mean `from before`?

Comment (aaranadev) [reply template-skeleton/template/package.dev.json:53]: In `npm install` the package.json is restructured single. This dependencie was before

Comment (alasarr) [reply template-skeleton/template/package.dev.json:53]: Got it

**Review by alasarr — APPROVED** [review]

**Comment by alasarr** [comment]
Need to fix conflicts

**Pull request merged**